zebra/rt_netlink.c

```diff
@@ -241,121 +241,120 @@ netlink_route_change_read_unicast (struct sockaddr_nl *snl, struct nlmsghdr *h,
       p.family = AF_INET;
       memcpy (&p.u.prefix4, dest, 4);
       p.prefixlen = rtm->rtm_dst_len;
+    }
```

Comment:
Missing initialization since the debug log below now uses this:
src_p.prefixlen = rtm->rtm_src_len;